hedera-node/src/main/java/com/hedera/services/ServicesMain.java

```diff
@@ -251,8 +251,8 @@ private void validateLedgerState() {
 			log.error("Unexpected total balance in ledger, nodeId={}!", ctx.id());
 			throw new IllegalStateException("Invalid total tinyBar float!");
 		}
-		if (ctx.nodeAccount() == null) {
-			throw new IllegalStateException("Unknown ledger account!");
+		if (!ctx.nodeInfo().isSelfZeroStake() && !ctx.nodeInfo().hasSelfAccount()) {
+			throw new IllegalStateException("Node is not zero-stake, but has no known account!");
```

Comment:
I don't see a change in the `ServicesMainTest` to reflect the exception error message. We should have expectation for that too as a strong code coverage.

Reply:
I'm comfortable with just verifying something is thrown here since it will cause the JVM to exit and we will get the stack trace...even if there was no message at all, it would still be easy to diagnose what happened.